start-all.sh

```diff
@@ -44,22 +46,40 @@ else
     PROMETHEUS_NAME=aprom-$PROMETHEUS_PORT
 fi
 
+if [ -z $NODES ]; then
+    NODES="127.0.0.1"
```

Comment:
I personally feel like that having two deployment methods is confusing (local conf file and -nodes switch). That's why I standardized everything on nodes, and defaulted to localhost if no node is passed.

For us it is great to be as flexible as possible but I think it may confuse users. @tzach what do you think ?